src/car.cpp

```diff
@@ -843,6 +848,7 @@ void Car::collided(int hit_triangle) {
 }
 
 float Car::get_collision_spin(int hit_triangle) {
+	Mix_PlayChannel(-1, m_crash, 0);

```

Comment:
bug I noticed while testing recently on Windows: if cars are honking, Roy's whistle sound effect won't play when you signal to a car. My guess is just that we're trying to play them on the same channel, so maybe just edit this to play on a different channel?

Reply:
Wasn't the issue. I was flooding the audio channels by accident, forget to protect the if statement.